pep-06XX-immortal-objects.rst

```diff
@@ -0,0 +1,342 @@
+PEP: 6XX
+Title: Immortal Objects, Using a Fixed Refcount
+Author: Eric Snow <ericsnowcurrently@gmail.com>, Eddie Elizondo <eelizondo@fb.com>
+PEP-Delegate: <TBD>
```

Comment:
For rendering it would be better to leave the `<TBD>` fields out whilst still undecided